htdocs/product/composition/card.php

```diff
@@ -365,7 +365,7 @@
 					$total+=$totalline;
 
 					print '<td class="right">';
-					print ($notdefined?'':($value['nb']> 1 ? $value['nb'].'x' : '').price($fourn_unitprice, '', '', 0, 0, -1, $conf->currency));
+					print ($notdefined?'':($value['nb']> 1 ? $value['nb'].'x' : '').price($totalline, '', '', 0, 0, -1, $conf->currency));
```

Comment:
Can you confirm it is totalline and not totalline / nb ?  
Because totalline contains already a multiplication by nb and the text is print "nb" x "unitprice".

Reply:
Good catch!